src/main/java/com/google/sps/servlets/InputServlet.java

```diff
@@ -63,11 +65,11 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     }
 
     TextAnalyser textAnalyser = new TextAnalyser(input_text);
-    Set<String> keywords = textAnalyser.getKeyWords();
-    keywords.add(user_location);
-    ImageSelection imageSelect = new ImageSelection(keywords);
+    Set<String[]> keywordSet = textAnalyser.getSetsOfKeyWords();
```

Comment:
Location of the user must be included, but I will pull request it later this day.